WeakAuras/BuffTrigger2.lua

```diff
@@ -1463,7 +1529,7 @@ end
 local function ScanAllGroup(time, matchDataChanged)
   -- We iterate over all raid/player unit ids here because ScanGroupUnit also
   -- handles the cases where a unit existence changes.
-  for unit in GetAllUnits("group") do
+  for unit in GetAllUnits("group", nil, true) do
```

Comment:
So the comment does not match what the function actually does.

And it turns out that's becasue the function is no longer used, so it can just be deleted.